docs/src/main/sphinx/connector/loki.md

```diff
@@ -174,3 +174,12 @@ FROM
     )
   )
 ;
+```
+
+The columns have the following names and types 
```

Comment:
We use `text` code block for a result example. Please take a look at https://trino.io/docs/current/sql/select.html.

Reply:
Please show result's example and avoid prepread statement.